src/components/virtualRepeat/virtualRepeater.scss

```diff
@@ -14,7 +14,8 @@ $virtual-repeat-scrollbar-width: 16px;
     left: 0;
     margin: 0;
     overflow-x: hidden;
-    overflow-y: auto;
+    overflow-y: scroll;
+    -webkit-overflow-scrolling: touch;
```

Comment:
If we're making this change, you don't need the override in the date picker css.